lib/http-store.js

```diff
@@ -0,0 +1,134 @@
+'use strict'
+const request = require('request');
+const util = require('util');
+const stormer = require('stormer');
+const assert = require('assert');
+const qs = require('qs');
+
+const Store = stormer.Store;
+const NotFoundError = stormer.errors.NotFoundError;
+
+/**
+ * @class HttpStore
+ * @extends Store
+ *
+ */
+class HttpStore extends Store {
+
+  /**
+   * Generates an HttpStore instance
+   *
+   * @constructor
+   * @memberof HttpStore
+   * @param {Object} requestClient - The request client to use
+   * @param {Object} options
+   * @return {HttpStore}
+   *
+   */
+  constructor(requestClient = request, options) {
+    assert(request, 'request is required');
+    assert(options, 'options are required');
+    super();
+    this._request = requestClient;
+    this._timeout = options.timeout || 1000;
+    this._baseUrl = options.baseUrl;
```

Comment:
instead of keeping the options and passing them every time, utilize [request.defaults()](https://github.com/request/request#requestdefaultsoptions)